lib/instrumentation/modules/aws-sdk/sns.js

```diff
@@ -0,0 +1,102 @@
+const constants = require('../../../constants')
+
+const TYPE = 'messaging'
+const SUBTYPE = 'sns'
+const ACTION = 'PUBLISH'
+const ACCESS_POINT = 'accesspoint'
+const PHONE_NUMBER = '<PHONE_NUMBER>'
+
+function getRegionFromRequest (request) {
+  return request && request.service &&
+        request.service.config && request.service.config.region
+}
+
+function getSpanNameFromRequest (request) {
+  const topicName = getDestinationNameFromRequest(request)
+  return `SNS ${ACTION} ${topicName}`
+}
+
+function getMessageContextFromRequest (request) {
+  return {
+    queue_name: getDestinationNameFromRequest(request)
```

Comment:
This should be `context.message.queue.name`

Reply:
✅